src/vm/amd64/gmscpu.h

```diff
@@ -65,26 +65,23 @@ struct MachState
     // because the context pointers below may point up to these 
     // fields.
     //
-    ULONG64 m_CaptureRdi;
-    ULONG64 m_CaptureRsi;
-    ULONG64 m_CaptureRbx;
-    ULONG64 m_CaptureRbp;
-    ULONG64 m_CaptureR12;
-    ULONG64 m_CaptureR13;
-    ULONG64 m_CaptureR14;
-    ULONG64 m_CaptureR15;
+    CalleeSavedRegisters m_Capture;
     
     // context pointers for preserved registers
-    PTR_ULONG64 m_pRdi;
-    PTR_ULONG64 m_pRsi;
-    PTR_ULONG64 m_pRbx;
-    PTR_ULONG64 m_pRbp;
-    PTR_ULONG64 m_pR12;
-    PTR_ULONG64 m_pR13;
-    PTR_ULONG64 m_pR14;
-    PTR_ULONG64 m_pR15;
+    struct {
```

Comment:
I think declaring an a separate struct for context pointers outside of this class (like you did for CalleeSavedRegisters) would make this peace of this of code a bit more readable and consistent with respect to m_Capture and m_Unwound.